companion.py

```diff
@@ -4,35 +4,40 @@
 import base64
 import csv
 import requests
-from http.cookiejar import LWPCookieJar	# No longer needed but retained in case plugins use it
+from http.cookiejar import LWPCookieJar  # No longer needed but retained in case plugins use it
```

Comment:
I need to make a note to look into turning "in case plugins use it" into an extra explicit include in setup_py instead.